tox.ini

```diff
@@ -40,8 +40,10 @@ select = E,W,F,C,N
 ignore = W503,E501
 
 [testenv:unit]
-deps = -r{toxinidir}/requirements.txt
-whitelist_externals = coverage
+deps = 
+  -r{toxinidir}/requirements.txt
```

Comment:
So you do not differentiate between requirements and dev requirements.

Reply:
dev reqs are basically test reqs, and with tox it can all be captured under deps property. speaking of which, I should update the docs